### DISCUSSION TITLE
v2raya doesn't load some websites and applications.

### DISCUSSION THREAD

**Discussion by alivkz** [v2rayA/v2rayA#1414] [General]
Hi
I have installed OpenWrt 22.03.4 and installed v2raya from (https://github.com/v2rayA/v2raya-openwrt). I set up v2raya and added a config to it. it worked well on some applications on iOS (like Telegram and Instagram) but some apps didn't load, like X, WhatsApp, Spotify.
Also, the YouTube app didn't load the videos's cover but it loaded videos perfectly!
Also, websites like [Instagram.com](http://instagram.com/), and [x.com](http://x.com/) didn't load!
my config works perfectly in the v2box app on iOS and loads all of the contents (it's protocol is VMESS).
How can I solve this problem??

this is my v2raya setup:
Transparent Proxy/System Proxy: On: Traffic splitting mode...
Transparent Proxy/System Proxy Implementation: redirect
Traffic Splitting Mode of Rule Port: RoutingA (
default: proxy

domain(ext:"iran.dat:ads")->block
domain(ext:"iran.dat:proxy")->proxy
domain(ext:"iran.dat:all")->direct
ip(geoip:ir)->direct)
)
Prevent DNS Spoofing: Advanced setting (
Domain Query Servers:
1.1.1.1 -> direct
8.8.8.8 -> direct
)
Special Mode: Off
TCPFastOpen: Keep default
Multiplex: Off
Automatically Update Subscriptions: Off
Mode when Update Subscriptions and GFWList: Follows Transparent Proxy/System proxy

**Comment by iamyegor** [comment]
Did you find the answer for your problem?

**Reply by bunbunpig** [reply]
which mode are you using? if in proxy mode, the system won't proxy some application (IG, whatsapp, TG...)
I've just try to make it in tproxy mode, and change the gateway of your PC, smart phone to your proxy IP address. please keep the default DNS at you router IP address.
aftrer these changes, all application will go through v2raya service, please try it, enjoy!

**Reply by iamyegor** [reply]
Some websites in Chrome were showing that I had a VPN IP address, while others were showing my original IP address, which was very inconvenient. So, I decided to use Nekoray, and it works perfect.

**Comment by hassanghgit** [comment]
Hi Ali, have you found any solution to open some sites including YouTube when setting v2ray client on router? I have same problem.

**Comment by hassanghgit** [comment]
I found the solution. When the outbound in client side is "ws" type (not tcp), v2ray client can well handle tcp and udp so all sites can work and be opened  smoothly. That's it. Enjoy the solution. Use v2ray servers that are configured as ws type in their inbound. Don't get it wrong: in server side, inbound should be configured as ws type and on the client side, the outbound should be configured as ws type, consequently.

**Comment by Mioingithub** [comment]
v2box uses the old kernel and you use the old vmess protocol. Maybe that's the problem. Change the protocol to vless+reality+raw, change the core to xray 25.10.15, and change/update the files.dat or use the ones supplied with the kernel, change the redirect mode to tproxy, system tun, or gvisor tun
